backend/src/main/java/hu/flowacademy/konyhatunder/service/RecipeService.java

```diff
@@ -36,8 +36,9 @@
         return recipeRepository.findAll();
     }
 
-    public Optional<Recipe> findById(String id) {
-        return recipeRepository.findById(id);
+    public Recipe findById(String id) {
```

Comment:
[think about] Nekem service szinten belefért volna az Optional, és a controller végezne rajta error handlinget, de ez is valid oopció.

Reply:
Rendben.